tests/unit/test_skiplist_graph.c

```diff
@@ -4,55 +4,43 @@
 #include "../../src/util/skiplist.h"
 #include "../../src/value.h"
 #include "../../src/graph/node.h"
+#include "../../src/index/index.h"
 
 char *words[] = {"foo",  "bar",     "zap",    "pomo",
                  "pera", "arancio", "limone", NULL};
 
 const char *node_label = "default_label";
 char *prop_key = "default_prop_key";
 
-int compareNodes(const void *p1, const void *p2) {
-  Node *a = (Node *)p1;
-  Node *b = (Node *)p2;
+// NOTE - these 'test*' functions are copies of those seen in 'index.c',
+// which can only be accessed from within the scope of index operations
```

Comment:
I like limiting the scope of the skiplist comparator functions, but needing to duplicate those functions here is admittedly rather ugly.